flink-connector-bigquery-common/src/main/java/com/google/cloud/flink/bigquery/common/utils/GoogleCredentialsSupplier.java

```diff
@@ -66,8 +65,18 @@ private static Credentials createCredentialsFromAccessToken(String accessToken)
 
     private static Credentials createCredentialsFromKey(String key) {
         try {
+            // Replaced BaseEncoding.base64() [of com.google.common.io.BaseEncoding]
+            // with Base64.getDecoder() since flink does not allow common.io methods
+            // to coexist and prefers usage of flink-shaded-guava methods instead.
+            // But that would cause dependency on flink, so replaced with java.utils.Base64
+            // Both support RFC-4648 (https://www.ietf.org/rfc/rfc4648.txt)
+            // Links:
+            // 1. BaseEncoding:
+            // https://guava.dev/releases/17.0/api/docs/com/google/common/io/BaseEncoding.html#base64()
+            // 2. Base64:
+            // https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html
             return GoogleCredentials.fromStream(
-                    new ByteArrayInputStream(BaseEncoding.base64().decode(key)));
+                    new ByteArrayInputStream(Base64.getDecoder().decode(key)));
```

Comment:
@yerramalli Please have a look at this replacement.

Reply:
Looks good, but please add unit tests to verify different types of encrypted keys.